cmd/genyaml/gen_kubectl_yaml.go

```diff
@@ -64,8 +64,7 @@ func main() {
 	// Set environment variables used by kubectl so the output is consistent,
 	// regardless of where we run.
 	os.Setenv("HOME", "/home/username")
-	// TODO os.Stdin should really be something like ioutil.Discard, but a Reader
-	kubectl := cmd.NewKubectlCommand(os.Stdin, ioutil.Discard, ioutil.Discard)
+	kubectl := cmd.NewKubectlCommand(genutils.NullReader, ioutil.Discard, ioutil.Discard)
```

Comment:
Instead of writing something specific, please just use `bytes.NewReader(nil)`. It does what I think is sensible: https://play.golang.org/p/7dxp9HUubHX

Reply:
I have `bytes.NewReader(nil)` to replace the original commit